README.md

```diff
@@ -1,2 +1,6 @@
-# mir-cv
-[WIP] Computer Vision
+# description
+[WIP] lower level CV algorithms (potentially implemented with `-betterC`), with optional C API headers
+
+Idea would be for DCV to be relying on mir-cv as dependency, providing higher level API for those lower level algorithms. See for eg #3 as first step towards that (see https://github.com/libmir/mir-cv/issues/4#issuecomment-367427761)
+
+But still unsure whether this separation is needed.
```

Comment:
I'd say this last line is not needed - It looks more like a criticism belonging in issues as an opener to a discussion, than closer line to a readme file. :) Would you mind removing it? But anyways, thank you for doing this - even though nothing has been going on, people should know the point of the repo.

Reply:
done, PTAL